templates/user/user-about.html

```diff
@@ -67,10 +130,122 @@ <h4>Rating History</h4>
+
+        $(document).ready(function () {
+            var active_tooltip = null;
+
+            function display_tooltip(where) {
+                if (active_tooltip !== null) {
+                    active_tooltip.removeClass(['tooltipped', 'tooltipped-e', 'tooltipped-w']).removeAttr('aria-label');
+                }
+                if (where !== null) {
+                    var day_num = parseInt(where.attr('data-day'));
+                    var tooltip_direction = day_num < 183 ? 'tooltipped-e' : 'tooltipped-w';
+                    where.addClass(['tooltipped', tooltip_direction])
+                         .attr('aria-label', where.attr('data-submission-activity'));
+                }
+                active_tooltip = where;
+            }
+
+            function install_tooltips () {
+                display_tooltip(null);
+                $('.activity-label').each(function () {
+                    var link = $(this);
+                    link.hover(
+                        function(e) {
+                            display_tooltip(link);
+                        },
+                        function(e) {
+                            display_tooltip(null);
+                        }
+                    );
+                });
+            }
+
+            var current_year = new Date().getFullYear();
+            var $div = $('#submission-activity');
+
+            function draw_contribution(year) {
+                $div.find('#submission-activity-table td').remove();
+                $div.find('#year').attr('data-year', year);
+                $div.find('#prev-year-action').css('display', year > metadata.min_year ? '' : 'none');
+                $div.find('#next-year-action').css('display', year < current_year ? '' : 'none');
+                var current_weekday = 0;
+                var start_day = new Date(year, 0, 1)
+                var end_day = new Date(year + 1, 0, 0);
+                if (year == current_year) {
+                    end_day = new Date();
+                    start_day = new Date(end_day.getFullYear() - 1, end_day.getMonth(), end_day.getDate() + 1);
+                    $div.find('#year').text("{{ _('past year') }}");
+                } else {
+                    $div.find('#year').text(year);
+                }
+                var days = [];
+                for (var day = start_day, day_num = 1; day <= end_day; day.setDate(day.getDate() + 1), day_num++) {
+                    var isodate = day.toISOString().split('T')[0];
+                    days.push({
+                        date: new Date(day),
+                        weekday: day.getDay(),
+                        day_num: day_num,
+                        activity: submission_activity[isodate] || 0,
+                    });
+                }
+
+                var sum_activity = days.map(obj => obj.activity).reduce((a, b) => a + b, 0);
+                // There doesn't seem to be a way to internationalize strings in JavaScript.
```

Comment:
/cc @quantum5

Reply:
Doesn't the code right under this comment already internationalize?

Reply:
I'm not sure if it's enough (ref. plurals).

Reply:
Fortunately, submissions are not fireplace pokers. Unfortunately, i18n should be done in JavaScript too, using the `gettext` function, as there are still pluralization issues.

Reply:
Done